data/custom_enums.py

```diff
@@ -0,0 +1,12 @@
+import enum
```

Comment:
Do you think it would be better to keep both `Enum` in `award.py`? All other enums are kept within the same file.

Reply:
issue resolved